.emacs.d/lisp/code/editorconfig-c.el

```diff
@@ -27,4 +27,28 @@ Debug warning is suppressed if SUPPRESS is non-nil."
 
 (kotct/editorconfig-check-for-core kotct/editorconfig-suppress-core-not-found-warning)
 
+(defvar kotct/warn-on-editorconfig-with-no-props
+  t
+  "If not false-y, warn when editing a file with no editorconfig properties
+applying, indicating that the user is stuck with Emacs' default style for the
+active buffer.")
+
+(defun kotct/check-editorconfig-props (props)
```

Comment:
i feel like this is likely testable, and therefore we should write a test for it. just create like `fakefile.fakefiletype` buffer and ensure that it warns.

Reply:
I'm finally working on these tests; should have them ready today.

Reply:
They have been pushed with the latest commit fb8d701.